.gitignore

```diff
@@ -80,6 +80,7 @@ scikits
 !np_datetime_strings.c
 *.c
 *.cpp
+*.pxi
```

Comment:
@jreback Do you know what is actually the advantage of calling a file `.pxi` instead of `.pyx`? Is it just a convention to easily know a file is included? As we eg also have files that are included in other files (eg "join.pyx" in "algos.pyx"), but those have just a `.pyx` extension